test/shared_test.go

```diff
@@ -392,6 +392,8 @@ func debugObjects(tc util.TestCtx) {
 	rows, err := conn.Query(tc.Context(), `
 		SELECT project, start_version, stop_version, path, mode, size, packed, (hash).h1, (hash).h2
 		FROM dl.objects
+		ORDER BY
+			project, start_version, stop_version, path
```

Comment:
It was much easier to view the debug data once it had been sorted

Reply:
Sounds good, but the formatting pattern is to keep the ORDER BY args on the same line as the ORDER BY (just as like SELECT)